packages/shared/src/contexts/PaymentContext.tsx

```diff
@@ -82,10 +83,15 @@ export const PaymentContextProvider = ({
   const [paddle, setPaddle] = useState<Paddle>();
   const { logSubscriptionEvent, isPlus } = usePlusSubscription();
   const logRef = useRef<typeof logSubscriptionEvent>();
-  logRef.current = logSubscriptionEvent;
+  const isExtension = checkIsExtension();
 
+  logRef.current = logSubscriptionEvent;
   // Download and initialize Paddle instance from CDN
   useEffect(() => {
+    if (isExtension) {
```

Comment:
I think we can do it like this so we won't have to add a dependency.
```suggestion
    if (checkIsExtension()) {
```